scripts/findDesignSystemVersions/index.ts

```diff
@@ -0,0 +1,125 @@
+import { request } from '@octokit/request';
+import Table from 'cli-table3';
+import chalk from 'chalk';
+import yargs from 'yargs';
+import dotenv from 'dotenv';
+
+const designSystemPackageNames = [
+  '@cmsgov/design-system',
+  '@cmsgov/ds-healthcare-gov',
+  '@cmsgov/ds-medicare-gov',
+  '@cmsgov/design-system-core',
```

Comment:
Is this the old, DS version? Do we still need to query for that?

Or are we querying to figure out if anyone is still using this?

Reply:
Yes, that's the old version. The list is pretty long still, so it might still be helpful to keep it around for a while. There's at least one there that I did not expect to see.